### PR TITLE
Feature: event modal interaction

### PR DESCRIPTION
**Pull request by edbrett**
Small PR to update conditional events modal:
- extract modal to partial
- set modal content to values from backend
- allow user to close event

additional: small css fixes and template updates

### CODE REVIEW

File: app/views/shared/_event-modal.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<% if @event %>
```
Comment (simaob) [inline app/views/shared/_event-modal.html.erb]: I think we should keep the _event_modal.html.erb (notice also the naming convention! snake_case yay!) inside `app/views/home/` as it's only used there, hence not shared!

File: app/views/shared/_event-modal.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <div class="c-events-modal">
+        <div class="event-container" style="background-image: url(<%= asset_path 'demo-event-bg.png' %>);">
+          <div class="js-btn-events-modal">
+            <svg class="icon icon-share">
+              <use xlink:href="#icon-close"></use>
+            </svg>
+          </div>
+          <div class="event-info">
+            <h3 class="text -title-single-alt"><%= @event.title %></h3>
+            <p class="text -body-single"><%= @event.description %></p>
```
Comment (simaob) [inline app/views/shared/_event-modal.html.erb]: Make sure to parse the text on the description, as we are using a WYSIWYG on the back!

Comment (pjosh) [inline app/views/shared/_event-modal.html.erb]: Could you use the real data source for the background? --> `@event.background_image`

**Comment by edbrett** [comment]
updated :)

**Comment by simaob** [comment]
Maybe @juancarlosalonso could provide a better placeholder image, for when one is not provided. I'll add as a task to PT!

**Pull request merged**